codewof/programming/content/en/inside-25-to-50/question.md

```diff
@@ -0,0 +1,7 @@
+# Inside 25 to 50
+
+The following function `inside_25_to_50(number, outside_mode)` takes a number `number` and a boolean `outside_mode` as arguments.
+The function should **return** `True` if the number is in the range 25 to 50 (inclusive) and `outside_mode` is `False`, otherwise it should return `False`.
+If `outside_mode` is `True`, the function should **return** `True` if the number is less than 25 or greater than 50. Otherwise it should return `False`.
```

Comment:
These two sentences swap their focus', consider changing one of them so that they describe the parameters in the same order; i.e. `return` then `number` then `outside_mode` or some other combination